include/account/account.h

```diff
@@ -4,16 +4,27 @@
 #include "address.h"
 #include <string>
 
+#ifndef DEFAULT_NONCE
+    #define DEFAULT_NONCE 0U
+#endif
+
+#define DEFAULT_BALANCE "0"
+
 class Account
 {
 public:
     explicit Account(Address address);
 
+    explicit Account(Address address, std::string balance, uint64_t const &nonce);
```

Comment:
👍

Reply:
Perhaps in the future we can implement a specialized class for handling the amount of tokens and use it instead of `std::string balance`.

Reply:
Yes, maybe something like here (a Balance class) in erdjs: https://github.com/ElrondNetwork/elrond-sdk-erdjs/blob/5c10d7d69dbc424464909398d0874684b59cb2c4/src/balance.ts#L27
For now, this is not a priority yet, but should be added in ERDCPP as well